docker/Dockerfile

```diff
@@ -48,7 +48,7 @@ ENV \
   GRPC_POLL_STRATEGY=epoll1
 
 # Copy and install dependencies first to leverage Docker layer caching.
-COPY /dist/requirements.txt /tmp/dist/requirements.txt
+COPY /dist/requirements-frozen.txt /tmp/dist/requirements-frozen.txt
```

Comment:
I think this is the reason for the cloudbuild fail `Step #2 - "runtime-image": error building image: error building stage: failed to optimize instructions: failed to get files used from context: copy failed: no source files specified` but I can't figure out where this `/dist` is....

Reply:
I think it's probably from python sdist which might try and include requirements.txt by default? Let's see.

Reply:
Well, we have one. manifest.in has `recursive-include ./ requirements*.txt` and to be double sure I made an sdist and it has it.

Reply:
I guess, let's try with cache=false.

Reply:
Ok, no. 

I don't see any reason why this can't just be changed to ./requirements-frozen.txt.

Reply:
Okay, so it's coming from 

```
pkginfo -f requires_dist --single --sequence-delim=! dist/*.whl | tr ! \\n > dist/requirements-frozen.txt
```

But still don't see any reason why this can't just be changed to ./requirements-frozen.txt.

Reply:
Huh back to square one... so I think `context` is a kaniko arg? And `copy failed: no source files specified` is from docker build.

Reply:
Ohhhhhhhhh.

```
.dockerignore
7:!/dist/requirements.txt
```